modules/ROOT/pages/how_to_contribute.adoc

```diff
@@ -0,0 +1,38 @@
+= How to Contribute to the ownCloud Documentation
+
+We welcome all helpful contributions to the ownCloud documentation, whether they’re reporting issues, adding new material, or writing tips and tricks. 
+
+If you wish to contribute to the official _Administration_, _User_, _Desktop Client_, and _Mobile Apps_ manuals, you must:
+
+* First, have a xref:https://github.com/[GitHub] account.
+* Second, fork the official repositories, write your material, and create a pull request. 
+
+The manuals are written using xref:https://docs.antora.org/antora/[Antora] and xref:https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[the AsciiDoc file format].
+
+* The Administration, Developer, and User manuals are in xref:https://github.com/owncloud/docs/[the owncloud/docs repository on GitHub].
+  Read xref:https://github.com/owncloud/docs/blob/master/docs/getting-started.md[the documentation] for instructions on setting up your build environment and becoming familiar with the file format and the build system that powers the documentation.
+* The documentation for the desktop sync client is in the doc directory of the Desktop Client repo. 
+  See xref:https://github.com/owncloud/client/blob/master/README.md[its README] for instructions.
+* The documentation of the Android app is in the user_manual directory of the Android App repo. 
+  See xref:https://github.com/owncloud/android/blob/master/README.md[its README] for instructions.
+* The documentation of the iOS app is in the user_manual directory of the iOS App repo. 
+  See xref:https://github.com/owncloud/ios/blob/master/README.md[its README] for instructions.
+
+== Reporting Documentation Errors
+
+If you find errors or omissions in the _Administration_, _Developer_, _User_, _Desktop Client_, and _Mobile Apps_ manuals, we encourage you to report them. 
+Please follow this template when reporting documentation errors:
+
+. First, search the respective repository for matching or similar issues.
+. If an issue is not available, open a new one in the respective repository.
+. Link to the error that you want to see fixed. 
+  Please be aware that there are multiple release versions of most of the manuals, so please be specific as to which version needs correcting, and if the issue is in the HTML or PDF version.
+. Quote the particular error.
+. Provide the correct solution (if you know it) and any links to supporting references. 
+  If you don't know the answer that is OK.
+. Provide screenshots if they add useful information.
+. Provide your web browser, its version, and your operating system & version.
+
+== Reporting Documentation Suggestions
+
+If you don't have a specific problem or contribution, but wish to offer a suggestion, open an issue in the appropriate repository.
```

Comment:
Proposal
``appropriate repository`` --> ``appropriate documentation repository``

Reply:
I have to disagree with this and the following comment, as the iOS, Android, and Desktop Client docs are all bundled in the source code. I see what you're saying, however.

Reply:
Means you think about a textual improvement or let it as it is 😄 ?